### DISCUSSION TITLE
General Question

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#105] [Q&A] [answered]
<img width="548" alt="Screenshot 2023-06-05 102546" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/414c1756-d1a5-4d81-be98-db5ba6849457">

I don't understand why sometimes you use "a" instead of typing "terms and conditions" before the {

**Comment by derekscholes** [comment, marked as answer]
This is a good question. From my understanding, *"a"* refers to the attribute tag **< a > < / a >**. Wherever this tag shows up in the code, the color parameter will be applied to the text within. Since the code only has one attribute tag which contains the text *"terms and conditions"*, that is the only text that is changed to the stated color. 

<img width="903" alt="Screenshot 2023-06-05 at 4 30 50 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132800749/cd9f04d9-9eee-46f2-ace0-c7149d8cd781">

Anytime you use CSS, the beginning label specifies the *"tag"* you want to change which changes the contents, and not the specific *"contents"* themselves. Does this make sense? If not, let me know.

**Reply by coopsand** [reply]
It is also important to note that while yes, `a` in before the opening braces correlates with the `<a>` tag, you can do the same thing with classes, as shown in building the cafe exercise from last week. To my understanding, using a `.` before the text in a styles file will indicate that the contents inside the braces will be applied to a specific class.
